axelrod/strategies/axelrod_first.py

```diff
@@ -538,49 +710,48 @@ def __init__(self, rounds_to_cooperate: int = 11) -> None:
         """
         Parameters
         ----------
-        rounds_to_cooperate: int, 10
+        rounds_to_cooperate: int
            The number of rounds to cooperate initially
         """
         super().__init__()
         self._rounds_to_cooperate = rounds_to_cooperate
         self.memory_depth = rounds_to_cooperate
 
     def strategy(self, opponent: Player) -> Action:
-        rounds = self._rounds_to_cooperate
-        if len(self.history) < rounds:
+        if len(self.history) < self._rounds_to_cooperate:
             return C
+        rounds = self._rounds_to_cooperate - 1
```

Comment:
Maybe we should hard-code `_rounds_to_cooperate` since setting it to 1 would cause a division by zero error

Reply:
Yeah I'm happy to do that. Looking back over `axelrod_first.py` I think we went a bit overboard on parametrising everything (if I remember correctly this was something I was intent on - my bad).